src/modules/claim/claims.controller.e2e.spec.ts

```diff
@@ -200,14 +204,14 @@ describe('ClaimsController', () => {
         expect(res.body.length).toEqual(1);
         expect(res.body[0]).toBeInstanceOf(Object);
         expect(res.body[0].issuedToken).toEqual(token);
-        expect(res.body[0].issuedAt).toHaveLength(10);
+        expect(res.body[0].issuedAt).not.toBeNaN();
         expect(res.body[0].subject).toEqual(requester);
       });
   });
 
   it('`/user/:did` should return only claim related to authenticated user', async () => {
-    const requester = Wallet.createRandom().address;
-    const foreignRequester = Wallet.createRandom().address;
+    const requester = randomDID();
+    const foreignRequester = randomDID();
```

Comment:
Weird that this test works still because it was just `address` before

Reply:
We do not have validation, so we can pass any string there and it will pass.

Reply:
Ah okay, I see, thanks